arviz/plots/jointplot.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.ticker import NullFormatter
+from .kdeplot import kdeplot
+from ..utils import trace_to_dataframe
+from .plot_utils import _scale_text
+
+
+def jointplot(trace, varnames=None, figsize=None, text_size=None, hexbin=False, gridsize='auto', skip_first=0,
+              joint_kwargs=None, marginal_kwargs=None):
+    """
+    Plot a scatter or hexbin of two variables with their respective marginals distributions.
+
+    Parameters
+    ----------
+
+    trace : Pandas DataFrame or PyMC3 trace
+        Posterior samples
+    varnames : list of variable names
+        Variables to be plotted, if None all variable are plotted
+    figsize : figure size tuple
+        If None, size is (8, 8)
+    text_size: int
+        Text size for labels
+    hexbin : Boolean
+        If True draws an hexbin plot
+    gridsize : int or (int, int), optional.
+        Only works when hexbin is True.
+        The number of hexagons in the x-direction. The corresponding number of hexagons in the
+        y-direction is chosen such that the hexagons are approximately regular.
+        Alternatively, gridsize can be a tuple with two elements specifying the number of hexagons
+        in the x-direction and the y-direction.
+    skip_first : int
+        Number of first samples not shown in plots (burn-in)
+    """
+    trace = trace_to_dataframe(trace, combined=True)[skip_first:]
+
+    if figsize is None:
```

Comment:
tuples can be used as default arguments.  also, thank you for allowing `figsize` as an argument!

Reply:
using `None` is consistent with other plots. For some plots figsize is not a predefined tuple of integers, but is computed taking into account the number of variables.